README improvements (#22651)
OpenRCT2/OpenRCT2: readme.md

---

# Contents
- 1. [Introduction](#1-introduction)
- 2. [Downloading the game (pre-built)](#2-downloading-the-game-pre-built)
- 3. [Building the game](#3-building-the-game)
  - 3.1 - [Building prerequisites](#31-building-prerequisites)
  - 3.2 - [Compiling and running](#32-compiling-and-running)
- 4. [Contributing](#4-contributing)
  - 4.1 - [Bug fixes](#41-bug-fixes)
  - 4.2 - [New features](#42-new-features)
  - 4.3 - [Translation](#43-translation)
  - 4.4 - [Graphics](#44-graphics)
  - 4.5 - [Audio](#45-audio)
  - 4.6 - [Scenarios](#46-scenarios)

> - 5. [License](#5-license)

Reviewer: Change this back. We use British English throughout.

# 2. Downloading the game (pre-built)

OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bought at either [Steam](https://store.steampowered.com/app/285330/RollerCoaster_Tycoon_2_Triple_Thrill_Pack/) or [GOG.com](https://www.gog.com/game/rollercoaster_tycoon_2). If you have the original RollerCoaster Tycoon and its expansion packs, you can [point OpenRCT2 to these](https://github.com/OpenRCT2/OpenRCT2/wiki/Loading-RCT1-scenarios-and-data) in order to play the original scenarios.

> [OpenRCT2.org](https://openrct2.org/downloads) offers pre-built builds and installers of the latest master and the develop branch. There is also a [launcher](https://github.com/IntelOrca/OpenLauncher/releases) available for Windows and Linux that will automatically update your build of the game so that you always have the latest version.

Reviewer: Pre-built builds sounds pleonastic. Also, this is a good opportunity to point to our own website, now that it has started listing the `develop` builds and the launcher

```suggestion
Our website, [OpenRCT2.io](https://openrct2.io/download) offers portable builds and installers of the latest `master` and the `develop` branch. There is also a [launcher](https://openrct2.io/download/launcher) available for Windows and Linux that will automatically update your build of the game so that you always have the latest version.
```

# 2. Downloading the game (pre-built)

OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bought at either [Steam](https://store.steampowered.com/app/285330/RollerCoaster_Tycoon_2_Triple_Thrill_Pack/) or [GOG.com](https://www.gog.com/game/rollercoaster_tycoon_2). If you have the original RollerCoaster Tycoon and its expansion packs, you can [point OpenRCT2 to these](https://github.com/OpenRCT2/OpenRCT2/wiki/Loading-RCT1-scenarios-and-data) in order to play the original scenarios.

[Our website](https://openrct2.io/download) offers portable builds and installers with the latest versions of the `master` and `develop` branches. There is also a [launcher](https://openrct2.io/download/launcher) available for Windows and Linux that will automatically update your build of the game so that you always have the latest version.

> Alternatively to using the launcher, for most Linux distributions, we recommend the [latest Flatpak stable release](https://flathub.org/apps/details/io.openrct2.OpenRCT2). When downloading from Flathub, you will always receive the latest updates regardless of which Linux distribution you use.

Reviewer 2: ```suggestion
Alternatively to using the launcher, for most Linux distributions, we recommend the [latest Flatpak release version](https://flathub.org/apps/details/io.openrct2.OpenRCT2). When downloading from Flathub, you will always receive the latest updates regardless of which Linux distribution you use.
```

Some Linux distributions offer native packages:

> * Arch Linux: [openrct2](https://archlinux.org/packages/extra/x86_64/openrct2/) stable release (`extra` repository) and, alternatively, [openrct2-git](https://aur.archlinux.org/packages/openrct2-git) (AUR)

Reviewer 2: ```suggestion
* Arch Linux: [openrct2](https://archlinux.org/packages/extra/x86_64/openrct2/) release version (`extra` repository) and, alternatively, [openrct2-git](https://aur.archlinux.org/packages/openrct2-git) (AUR)
```

</details>

<details>

>   <summary>Linux pre-requisites</summary>

Reviewer 2: https://dictionary.cambridge.org/dictionary/english-polish/prerequisite

---

> # 5. License

Reviewer: Please change back.

> **OpenRCT2**, an open-source re-implementation of RollerCoaster Tycoon 2.
> 
> Copyright (C) 2014 OpenRCT2 contributors
> 
> This program is free software: you can redistribute it and/or modify
> it under the terms of the GNU General Public License as published by
> the Free Software Foundation, either version 3 of the License, or
> (at your option) any later version.
> 
> This program is distributed in the hope that it will be useful,
> but WITHOUT ANY WARRANTY; without even the implied warranty of
> MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
> GNU General Public License for more details.
> 
> You should have received a copy of the GNU General Public License
> along with this program. If not, see <http://www.gnu.org/licenses/>.

Reviewer: There is no need to copy the licence text, but we should point using to the licence file:

```suggestion
**OpenRCT2** is licensed under the GNU General Public License version 3. See the `licence.txt` file for more details.
```

---

# 5. Licence

> **OpenRCT2** is licenced under the GNU General Public License version 3 or (at your option) any later version. See the [`licence.txt`](licence.txt) file for more details.

Reviewer: ```suggestion
**OpenRCT2** is licensed under the GNU General Public License version 3 or (at your option) any later version. See the [`licence.txt`](licence.txt) file for more details.
```

OK, I should have explained this a bit more. British English uses -se for the verb and -ce for the noun, while US English uses -se for both.
